configuration/pih/petl/jobs/sql/extractions/mch_kellogg_data.sql

```diff
@@ -0,0 +1,333 @@
+-- set @locale = 'en';  -- uncomment for testing
+
+set sql_safe_updates = 0;
+set session group_concat_max_len = 100000;
+
+select program_id into @mchProgram from program where uuid = '41a2715e-8a14-11e8-9a94-a6cf71072f73';
+select concept_name(concept_from_mapping('PIH','YES'),@locale) into @yes;
+select concept_name(concept_from_mapping('PIH','NO'),@locale) into @no;
+select name into @socioEncName from encounter_type where uuid = 'de844e58-11e1-11e8-b642-0ed5f89f718b';
+select program_workflow_id into @mchWorkflow from program_workflow where uuid = '41a277d0-8a14-11e8-9a94-a6cf71072f73';
+set @past_med_finding = concept_from_mapping('PIH','10140');
+select encounter_type_id into @ob_gyn_enc_id from encounter_type where uuid = 'd83e98fd-dc7b-420f-aa3f-36f648b4483d';
```

Comment:
is there no encounter_type(uuid) function?

Reply:
oh yeah...there is.  will use that.